servicebox-app/src/main/java/io/halkyon/resource/page/ClaimResource.java

```diff
@@ -50,6 +44,19 @@ public TemplateInstance list() {
         return showList(Claim.listAll()).data("all", true);
     }
 
+    @GET
+    @Produces(MediaType.TEXT_HTML)
+    @Path("/filter")
+    public Object filter(@QueryParam("name") String name, @QueryParam("servicerequested") String serviceRequested, @HeaderParam("HX-Request") boolean hxRequest) {
+        List<Claim> claims = Claim.getClaims(name, serviceRequested);
+        if (hxRequest) {
+            return Templates.Claims.table(claims);
+        } else {
+            // return showList(Claim.getClaims(name, serviceRequested)).data("all",false);
+            return null;
```

Comment:
I don't understand this logic. Why do we only return the data if there is the `HR-Request` header present?

Reply:
We should return something for non HTMX calls but I still don't know what should return: full HTML page including the claims, only the claims table, etc like also if we should we return HTML/JSON or another format

Reply:
But we're returning HTML, regardless if the client is using htmx or not, so we should always return the table.